exporter/collector/googlemanagedprometheus/monitoredresource_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 Google LLC
```

Comment:
Unit testing is useful, but it may be valuable to run GMP collector and GMP exporter side-by-side in the same testing environment and compare the exported metrics to see whether the configured resource labels for GMP exporter match expectations (and resemble the GMP collector where it makes sense)

Reply:
Sounds good.  We can definitely do that.  If there are particular setups you would like me to test, let me know.